src/components/FindMovie/FindMovie.tsx

```diff
@@ -16,42 +56,59 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              className="input"
+              value={query}
+              onChange={handleQuery}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {error && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={
+                isLoading
+                  ? 'button is-light is-loading'
+                  : 'button is-light'
+              }
+              disabled={!query}
             >
               Find a movie
             </button>
           </div>
 
-          <div className="control">
-            <button
-              data-cy="addButton"
-              type="button"
-              className="button is-primary"
-            >
-              Add to the list
-            </button>
-          </div>
+          {movie && (
+            <div className="control">
+              <button
+                data-cy="addButton"
+                type="button"
+                className="button is-primary"
+                onClick={() => {
+                  handleClear();
```

Comment:
You are clearing the form before adding the movie to the list. When handleAddMovie is asynchronous this could lead to race conditions where the movie is not added. It would be safer to clear the form after the movie has been added successfully.